plugins/module_utils/gitlab.py

```diff
@@ -110,3 +110,14 @@ def gitlab_authentication(module):
             GitLab remove Session API now that private tokens are removed from user API endpoints since version 10.2." % to_native(e))
 
     return gitlab_instance
+
+
+def preprocessing_returned_variables(gitlab_keys):
+    # pop properties we don't know
+    existing_variables = [x.attributes for x in gitlab_keys]
+    KOWN = ['key', 'value', 'masked', 'protected', 'variable_type', 'environment_scope']
+    for item in existing_variables:
+        for key in list(item.keys()):
+            if key not in KOWN:
+                item.pop(key)
+    return existing_variables
```

Comment:
I think you have a typo here, and maybe `filter_` would be a bit clearer on what the helper is doing?
```suggestion
def filter_returned_variables(gitlab_keys):
    # pop properties we don't know
    existing_variables = [x.attributes for x in gitlab_keys]
    KNOWN = ['key', 'value', 'masked', 'protected', 'variable_type', 'environment_scope']
    for item in existing_variables:
        for key in list(item.keys()):
            if key not in KNOWN:
                item.pop(key)
    return existing_variables
```